src/ipatool.ts

```diff
@@ -0,0 +1,195 @@
+type SearchResult = {
+  level: string;
+  count: number;
+  apps: App[];
+  time: string;
+};
+
+type App = {
+  id: number;
+  bundleID: string;
+  name: string;
+  version: string;
+  price: number;
+};
+
+const bundleIdentifierGenerator: Fig.Generator = {
+  script: (context) => {
+    const identifier = context.at(3);
```

Comment:
Keeping `3` as a constant may result in wrong suggestions, instead the last item of the context is always the current argument, see https://fig.io/docs/reference/generator/contextual#writing-a-contextual-script-function
```suggestion
    const identifier = context[context.length - 1];
```